ext/CatalystGraphMakieExtension.jl

```diff
@@ -1,11 +1,13 @@
 module CatalystGraphMakieExtension
 
 # Fetch packages.
-using Catalyst, GraphMakie
+using Catalyst, GraphMakie, Graphs 
 import Catalyst: lattice_plot, lattice_animation, extract_vals
-import Graphs: AbstractGraph, SimpleGraph
+import Graphs: AbstractGraph, SimpleGraph, SimpleDiGraph, SimpleEdge, src, dst, ne, nv 
+import Catalyst: speciesreactiongraph, incidencematgraph
```

Comment:
Some of these imports were there before I believe. Though I thought it'd make it easier to not have to write out Graphs.SimpleEdge, etc. too much in the code since some of these names aren't exported by default, but I can just have them be qualified in the code, no issue there

Reply:
Just use `using` then. (We should eventually get rid of import everywhere anyways I think, but that is not high priority).